infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/LogReplicationEventMetadata.java

```diff
@@ -13,10 +13,10 @@ public class LogReplicationEventMetadata {
     private static final UUID NIL_UUID = new UUID(0,0);
 
     /*
-     * Represents the request/event Id that preceded this event.
+     * Represents the request ID of snapshot_sync or log_entry_sync state
      *
-     * This is used to correlate the event with the state in which it was originated.
-     * For example, a trimmed exception from state A vs. a trimmed exception from state B.
+     * This is used to correlate the sync and the event.
+     * For example, a snapshot sync ID 1 , event E1 and a snapshot sync ID2 with event E2
```

Comment:
as there is no functional correlation between the event and request ids, it is better to reword this explanation and remove all references to 'event' IMO

Reply:
The comment is stating how the requestID is used.
Its essentially used to check if the incoming event is for the same sync or not. I tried to convey this in the comment.

I can include a line as to what "requestId" is.

Would that be enough to paint the picture of why its needed and how its used?

Reply:
Sorry, the comment was incomplete. I have completed it and removed the example as that did not make much sense.